src/forms/EditListingAudioForm/EditListingAudioForm.example.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable no-console */
+import EditListingAudioForm from './EditListingAudioForm'
+
+export const Empty = {
+  component: EditListingAudioForm,
+  props: {
+    initialValues: { country: 'US', images: [] },
+    stripeConnected: false,
+    onImageUpload: (values) => {
+      console.log(`onImageUpload with id (${values.id}) and file name (${values.file.name})`)
```

Comment:
These logs can stay if you need them but if things are working well we will eventually remove, of course.

Reply:
This is existing code for the examples